indra/newview/llvlcomposition.cpp

```diff
@@ -42,6 +42,8 @@
 #include "llviewercontrol.h"
 
 
+extern LLColor4U MAX_WATER_COLOR;
```

Comment:
seems like this declaration belongs in llsurface.h (and a similar extern declaration in llworld.cpp should probably be consolidated there along with it.)

possibly we also want this to be `const` as well?

Reply:
In hindsight, I probably should have checked the coding standard for conventions on global variables: https://wiki.secondlife.com/wiki/Coding_standard

It looks like static class members are the future. My understanding is that the gVariableName convention is superseded by `LLSimpleton`. There are some possible exceptions though, like indra_constants.cpp .

Will keep this in mind for the future.

Side note: Before seeing `MAX_WATER_COLOR` in llworld.cpp, I did not know was possible to use extern in this way.